src/applications/bmqtool/bmqtool.m.cpp

```diff
@@ -434,9 +434,6 @@ int main(int argc, const char* argv[])
     // Make TimeUtil thread-safe by calling initialize
     bsls::TimeUtil::initialize();
 
-    // Prepare the UriParser regexp
-    bmqt::UriParser::initialize();
```

Comment:
Note that there were no `shutdown` in `bmqtool`.
Also, there were no `initialize`/`shutdown` in our tutorial clients.